### PR TITLE
Add axisregistry opentypespec beziers font-v	stringbrewer booleanoperations	fontmath ufo2ft cffsubr fontparts ufolib2 cu2qu gflanguages ufolint defcon openstep-plist uharfbuzz dehinter opentype-sanitizer vharfbuzz

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/axisregistry/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+    - setuptools-scm >=4,<6.1
```
Comment (ocefpaf) [inline recipes/axisregistry/meta.yaml:22]:
```suggestion
    - setuptools-scm >=4,<6.1
```

File: recipes/fontparts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+    - setuptools-scm ==7.0.5
```
Comment (ocefpaf) [inline recipes/fontparts/meta.yaml:22]:
```suggestion
    - setuptools-scm ==7.0.5
```

File: recipes/gflanguages/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+    - setuptools-scm >=4,<6.1
```
Comment (ocefpaf) [inline recipes/gflanguages/meta.yaml:22]:
```suggestion
    - setuptools-scm >=4,<6.1
```

File: recipes/opentypespec/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentypespec-{{ version }}.tar.gz
+  sha256: 03621635fed0d04345fa7b97837de31f992fa6619aeefcca0e75e342248dbde2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # Windows cannot decode the 'charmap' codec byte 0x9d in position 721
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/opentypespec/meta.yaml:16]: Sometimes that can be noarch and installation will be fine even if the build fails.

Comment (jan-janssen) [reply recipes/opentypespec/meta.yaml:16]: @ocefpaf Sure - is it okay if I do this once the pull request is merged? I just want to make sure that all builds are working, as there are so many in this one pull request.

**Comment by jan-janssen** [comment]
@ocefpaf From my perspective this is ready to be merged.

**Pull request merged**